src/dosext/mfs/mfs.c

```diff
@@ -454,6 +454,29 @@ static int GetCDSInDOS(uint8_t dosdrive, cds_t *cds)
   return 1;
 }
 
+static int drive_from_name(const char *name, int simpledriveonly, int *drive)
```

Comment:
That looks too messy.
`simpledriveonly`, plus `LPT` is not accountable
in most functions (i.e. if you create dir named "LPT476"
I wonder if you need to apply `PRINTER_BASE_DRIVE` -
likely not). I think its better not to combine something
inherently incompatible, or you'll implement another
`select_drive()`. :)

Reply:
yep, it didn't feel quite right when doing it!